librosa/feature/spectral.py

```diff
@@ -1163,6 +1164,104 @@ def logfsgram(y=None, sr=22050, S=None, n_fft=4096,
     return cq_basis.dot(S)
 
 
+@cache
+def tonnetz(y=None, sr=22050, chromagram=None, norm=np.inf):
+    '''Computes the tonal centroid features (tonnetz).
+
+    Parameters
+    ----------
+    y : np.ndarray [shape=(n,)] or None
+        Audio time series.
+
+    sr : int > 0 [scalar]
+        sampling rate of `y`
+
+    chromagram : np.ndarray [shape=(n_chroma, t)] or None
+        Normalized energy for each chroma bin at each frame.
+
+        If `None`, a cqt chromagram is performed.
+
+    norm : float or None
+        Column-wise normalization.
+        See `librosa.util.normalize` for details.
+
+        If `None`, no normalization is performed.
+
+    Returns
+    -------
+    ton : np.ndarray [shape(6, t)]
+        Tonal centroid features for each frame.
+        Tonnetz dimensions:
```

Comment:
Yeah, I actually thought about that.

'x' and 'y' represent the two dimensions of a circle. The tonal centroids are typically represented by a [torus](http://33.media.tumblr.com/c8a40bdbd7815016fd5ce6dc8983b8c2/tumblr_motw9jlB6I1qfadvwo1_r1_400.gif), which can be seen as three circles: fifths, minor, and major.

I believe there's no standard way of labeling the axis. In my dissertation I used (LaTeX syntax) 5_x, 5_y,  m3_x, m3_y, M3_x, M3_y. If you fancy this, I can add it to `librosa.display.specshow`.